import path from 'path';
export const getPage0 = async(req, res, next) => {
    try {

        res.sendFile('pages/page-0.html', { root: "public" });
    } catch (error) {
        next(createHttpError(500, 'Internal Server Error'));
    }
}


export const getPage1 = async(req, res, next) => {
    try {

        res.sendFile('pages/page-1.html', { root: "public" });
    } catch (error) {
        next(createHttpError(500, 'Internal Server Error'));
    }
}
export const getPage2 = async(req, res, next) => {
    try {

        res.sendFile('pages/page-2.html', { root: "public" });
    } catch (error) {
        next(createHttpError(500, 'Internal Server Error'));
    }
}

export const getPage3 = async(req, res, next) => {
    try {

        res.sendFile('pages/page-3.html', { root: "public" });
    } catch (error) {
        next(createHttpError(500, 'Internal Server Error'));
    }
}

export const getPage4 = async(req, res, next) => {
    try {

        res.sendFile('pages/page-4.html', { root: "public" });
    } catch (error) {
        next(createHttpError(500, 'Internal Server Error'));
    }
}

export const getPage5 = async(req, res, next) => {
    try {

        res.sendFile('pages/page-5.html', { root: "public" });
    } catch (error) {
        next(createHttpError(500, 'Internal Server Error'));
    }
}

export const getPage6 = async(req, res, next) => {
    try {

        res.sendFile('pages/page-6.html', { root: "public" });
    } catch (error) {
        next(createHttpError(500, 'Internal Server Error'));
    }
}
export const getPage7 = async(req, res, next) => {
    try {

        res.sendFile('pages/page-7.html', { root: "public" });
    } catch (error) {
        next(createHttpError(500, 'Internal Server Error'));
    }
}

export const getPage8 = async(req, res, next) => {
    try {

        res.sendFile('pages/page-8.html', { root: "public" });
    } catch (error) {
        next(createHttpError(500, 'Internal Server Error'));
    }
}

export const getPage9 = async(req, res, next) => {
    try {

        res.sendFile('pages/page-9.html', { root: "public" });
    } catch (error) {
        next(createHttpError(500, 'Internal Server Error'));
    }
}

export const getPage10 = async(req, res, next) => {
    try {

        res.sendFile('pages/page-10.html', { root: "public" });
    } catch (error) {
        next(createHttpError(500, 'Internal Server Error'));
    }
}

export const getPage11 = async(req, res, next) => {
    try {

        res.sendFile('pages/page-11.html', { root: "public" });
    } catch (error) {
        next(createHttpError(500, 'Internal Server Error'));
    }
}